.travis.yml

```diff
@@ -16,13 +16,11 @@ jobs:
   - name: Minikube
     script:
     - scripts/setup-minikube-for-linux.sh
-    - make minikube-test
-    env: USE_LATEST_IMAGES=false
+    - make minikube-test CHARTS=enterprise-suite
```

Comment:
In my previous incarnation of this file, I had one call to `make minikube-test` that let the `Makefile` run that on all the projects.  I've changed this to invoke `make minikube-test` on `enterprise-suite` and `enterprise-suite-latest` individually.  This is more in line with what you had and what I think you're looking for.  Works for me.  I agree that it clarifies the intent.